setup:
    prefix: 'j2s7s300_driver'
    model_filename: "jaco_dynamics"
    object_centers: {'HUMAN_CENTER': [-0.6,-0.55,0.0], 'LAPTOP_CENTER': [-0.7929,-0.1,0.0]}
    feat_list: ["efficiency", "table", "coffee"]
    feat_weights: [1.0,0.0,1.0]
    start: [104.2, 151.6, 183.8, 101.8, 224.2, 216.9, 225.0]
    goal: [210.8, 101.6, 192.0, 114.7, 222.2, 246.1, 322.0]
    goal_pose: [-0.46513, 0.29041, 0.69497]
    T: 20.0
    timestep: 0.5

planner:
    # These settings have been tuned for trajopt planner.
    type: "trajopt"
    max_iter: 50
    num_waypts: 5

controller:
    # These settings have been tuned for PID controller.
    type: "pid"
    p_gain: 50.0
    i_gain: 0.0
    d_gain: 20.0
    epsilon: 0.10
    max_cmd: 40.0

